src/core/src/main/java/org/apache/jmeter/testelement/TestPlan.java

```diff
@@ -83,11 +86,23 @@ public void prepareForPreCompile()
 
     /**
      * Fetches the functional mode property
-     *
+     * Could be change for no gui test with jmeter property : PROP_FUNCTIONAL_MODE
      * @return functional mode
      */
     public boolean isFunctionalMode() {
-        return getPropertyAsBoolean(FUNCTIONAL_MODE);
+        boolean bFunctionalModeDefault = getPropertyAsBoolean(FUNCTIONAL_MODE);
```

Comment:
For a method-local variable I would not use *b* as a prefix to mark the variable as a boolean. If the method is so long or complex, that I can't remember it is a boolean, it should be shortened :)
For global variables I would not use the prefix, as Java has really good IDE support, which shows those information already ;)

Reply:
Changed with proposal modification, change bFunctionalModeDefault  to functionalModeDefault